DragaliaAPI/DragaliaAPI/Services/Game/StoryService.cs

```diff
@@ -343,4 +358,49 @@ AtgenBuildEventRewardEntityList reward
 
         return questReward;
     }
+
+    private static List<Present> GetChapter10PresentList()
+    {
+        Dictionary<Materials, int> materialCounts =
+            new()
+            {
+                { Materials.WindOrb, 76 },
+                { Materials.StormOrb, 1 },
+                { Materials.WaterOrb, 86 },
+                { Materials.StreamOrb, 3 },
+                { Materials.DelugeOrb, 1 },
+                { Materials.FlameOrb, 116 },
+                { Materials.BlazeOrb, 7 },
+                { Materials.InfernoOrb, 2 },
+                { Materials.LightOrb, 270 },
+                { Materials.RadianceOrb, 15 },
+                { Materials.RefulgenceOrb, 3 },
+                { Materials.ShadowOrb, 66 },
+                { Materials.Talonstone, 87 },
+                { Materials.LightMetal, 18 },
+                { Materials.IronOre, 25 },
+                { Materials.Granite, 10 },
+                { Materials.FiendsClaw, 25 },
+                { Materials.FiendsHorn, 10 },
+                { Materials.BatsWing, 25 },
+                { Materials.AncientBirdsFeather, 10 },
+                { Materials.DyrenellAes, 2520 }
+            };
+
+        List<Present> presents =
+            new() { new Present(PresentMessage.Chapter10Clear, EntityTypes.HustleHammer, 0, 350) };
+        foreach ((Materials material, int count) in materialCounts)
+        {
+            presents.Add(
+                new Present(
+                    PresentMessage.Chapter10Clear,
+                    EntityTypes.Material,
+                    (int)material,
+                    count
+                )
+            );
+        }
```

Comment:
I would recommend taking a more functional approach and using LINQ, it's a bit easier to read and the modern standard for C#.

Instead of accumulating into a list you can do

```cs
List<Present> presents =
    new() { new Present(PresentMessage.Chapter10Clear, EntityTypes.HustleHammer, 0, 350) };

IEnumerable<Present> addedPresents  = materialCounts.Select((key, value) => 
       // Select: transform each element of the sequence
      new Present(
            PresentMessage.Chapter10Clear,
            EntityTypes.Material,
            (int)material,
            count
      )
); 

// The above IEnumerable is lazily evaluated, and realized when added to the list
presents.AddRange(addedPresents);
```

And this can be tidied up even further with C# 12 collection expressions:

```cs
List<Present> presents =
[
    new Present(PresentMessage.Chapter10Clear, EntityTypes.HustleHammer, 0, 350),
    .. materialCounts.Select(
        (key, value) =>
            new Present(PresentMessage.Chapter10Clear, EntityTypes.Material, (int)material, count)
    )
];
```